<ui version="4.0" >
 <class>searchDialogForm</class>
 <widget class="QWidget" name="searchDialogForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>133</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Find Object</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="sizeConstraint" >
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="BarFrame" name="TimeBar" >
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="autoFillBackground" >
      <bool>false</bool>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>4</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Find Object</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="closeButton" >
        <property name="maximumSize" >
         <size>
          <width>17</width>
          <height>17</height>
         </size>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet" >
         <string notr="true" >QToolButton {
color: white;
font-weight: bold;
background: None;
border: 1px solid #2e2e36;
  border-radius: 2px;
}

QToolButton:pressed {
color: darkgray;
}</string>
        </property>
        <property name="text" >
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QLineEdit" name="lineEditSearchSkyObject" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="palette" >
           <palette>
            <active>
             <colorrole role="Base" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>169</red>
                <green>169</green>
                <blue>169</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Base" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>169</red>
                <green>169</green>
                <blue>169</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Base" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>169</red>
                <green>169</green>
                <blue>169</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
          <property name="font" >
           <font/>
          </property>
          <property name="focusPolicy" >
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="autoFillBackground" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonGotoSearchSkyObject" >
          <property name="minimumSize" >
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="focusPolicy" >
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text" >
           <string>Go</string>
          </property>
          <property name="checkable" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="CompletionLabel" name="completionLabel" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="MinimumExpanding" hsizetype="Preferred" >
          <horstretch>35</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string/>
        </property>
        <property name="scaledContents" >
         <bool>false</bool>
        </property>
        <property name="alignment" >
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
        <property name="wordWrap" >
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags" >
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BarFrame</class>
   <extends>QFrame</extends>
   <header>Dialog.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CompletionLabel</class>
   <extends>QLabel</extends>
   <header>SearchDialog.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditSearchSkyObject</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
